### PR TITLE
Add proxy params to BigQuery DataSource

### PR DESCRIPTION
**Pull request by abhishoya-gs**
## Summary
Adding the proxy parameters to BigQuery DataSource Specification 
Engine PR: https://github.com/finos/legend-engine/pull/618
Pure PR: https://github.com/finos/legend-pure/pull/499

## How did you test this change?

- [X] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

**Review by MauricioUyaguari — CHANGES_REQUESTED** [review]
Please update grammar roundtrip tests: https://github.com/finos/legend-studio/blob/master/packages/legend-manual-tests/src/__tests__/roundtrip-grammar/cases/relational-connection.pure

File: .changeset/swift-peas-juggle.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-studio': patch
+---
```
[the remark this answers is not in the document]
Comment (abhishoya-gs) [reply .changeset/swift-peas-juggle.md:4]: Added a new changeset in this [commit](https://github.com/finos/legend-studio/pull/1010/commits/f464e34d16adef0a3da2b1185e893c193540bde0)

File: packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx [hunk 2]
```diff
@@ -816,6 +818,28 @@ const BigQueryDatasourceSpecificationEditor = observer(
             )
           }
         />
+        <ConnectionEditor_StringEditor
+          isReadOnly={isReadOnly}
+          value={sourceSpec.proxyHost}
+          propertyName="proxy host"
+          update={(value: string | undefined): void =>
```
[the remark this answers is not in the document]
Comment (abhishoya-gs) [reply packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx:826]: Added in this [commit](https://github.com/finos/legend-studio/pull/1010/commits/f464e34d16adef0a3da2b1185e893c193540bde0)

File: .changeset/metal-bottles-happen.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+'@finos/babel-preset-legend-studio': patch
+'@finos/eslint-plugin-legend-studio': patch
+'@finos/legend-application': patch
```
Comment (gayathrir11) [inline .changeset/metal-bottles-happen.md:4]: You don't need this file. We only add changesets to the `packages` we make changes too. Please delete this file and add the desciption to the other changeset you added

Comment (abhishoya-gs) [reply .changeset/metal-bottles-happen.md]: Added a new single changeset for the three packages

File: .changeset/stupid-turtles-invite.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio': patch
+---
```
Comment (MauricioUyaguari) [inline .changeset/stupid-turtles-invite.md:5]: Let's just keep `@finos/legend-graph` and `@finos/legend-studio` here. 
We can add an empty changeset with no message that has 
`@finos/legend-manual-tests`

**Review by MauricioUyaguari — CHANGES_REQUESTED** [review]

File: .changeset/stupid-turtles-invite.md (hunk 4 above)
Comment (abhishoya-gs) [reply .changeset/stupid-turtles-invite.md:4]: Changed [here](https://github.com/finos/legend-studio/pull/1010/commits/726df0dc39fe0206abcaf4a5f5c4eb4aeac086b4)

**Comment by MauricioUyaguari** [comment]
@abhishoya-gs  is this ready for review again. If so could you move it out of the draft state. Thanks

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**